engine/tests/phpunit/Elgg/ActionsServiceTest.php

```diff
@@ -344,7 +344,7 @@ public function testCanNotExecuteUnregisteredAction() {
 
 		$this->assertInstanceOf(ErrorResponse::class, $result);
 		$this->assertEquals(ELGG_HTTP_NOT_IMPLEMENTED, $result->getStatusCode());
-		$this->assertEquals(elgg_echo('actionundefined'), $result->getContent());
+		$this->assertEquals(elgg_echo('actionundefined', ['unregistered']), $result->getContent());
```

Comment:
Oddly these tests work in 2.x I think because elgg_echo() in PHPUnit was broken until this change.